### PR TITLE
Adds sticky nav

### PR DESCRIPTION
**Pull request by savance-ford**
:green_heart: 

ready to go!!!!!!!!

### CODE REVIEW

File: css/style.css [hunk 1]
```diff
@@ -14,6 +14,40 @@ body {
...
+    left: 0;
+    top: 0;
+    z-index: 100;
+    border-top: 0;
+}
+
+ul {
+  margin: 0px;
+  padding: 0px;
+}
```
Comment (whazzmaster) [inline css/style.css:38]: Small style nitpick: include spaces between each outer block.

File: index.html [hunk 2]
```diff
@@ -5,7 +5,47 @@
   <meta charset="UTF-8">
   <title>LMHT</title>
   <link type='text/css' href='css/style.css' rel='stylesheet'/>
+  <script src="https://ajax.googleapis.com/ajax/libs/jquery/1.8.3/jquery.min.js" type="text/javascript"></script>
+<script src="js/jquery-scrolltofixed-min.js" type="text/javascript"></script>
+
+<script type="text/javascript">
+
+
+    $(document).ready(function() {
```
Comment (whazzmaster) [inline index.html]: You may want to consider moving this to its own file and including via a script tag, but if this just the first stab and you're going to iterate on it I'm ok with leaving it here.

File: index.html [hunk 3]
```diff
@@ -29,6 +69,20 @@
...
+        <nav class="nav">
+          <ul>
+            <li><textarea rows="2" cols="40">
+                place holder for comobo box
+                </textarea>
+            </li>
+          </ul>
+        </nav>
+      
+
```
Comment (whazzmaster) [inline index.html]: Lots of random whitespace included in the diff- you could pare this down.

**Comment by whazzmaster** [comment]
Just a little stylistic feedback.

**Comment by LipGlossary** [comment]
When the nav goes sticky, the remainder of the page "jumps" up behind it. It probably needs some padding to counteract the removal of the nav from the flow of the page.

**Comment by savance-ford** [comment]
Thanks for the Feedback! :+1:

**Comment by savance-ford** [comment]
@LipGlossary Hey can you elaborate a bit more on that?

File: css/style.css [hunk 4]
```diff
@@ -14,6 +14,58 @@ body {
...
 }
 
+
+/*sitckybar*/
+
+
+
+
+
+
```
Comment (bigtiger) [inline css/style.css]: Whats with all the whitespace ^^^?

File: css/style.css [hunk 5]
```diff
@@ -14,6 +14,58 @@ body {
...
+nav ul li {
+  list-style: none;
+  margin-left: 600px;
+}
+
+
+
+/*=============*/
+
+
```
Comment (bigtiger) [inline css/style.css:56]: Whitespace?

File: index.html [hunk 6]
```diff
@@ -5,7 +5,20 @@
   <meta charset="UTF-8">
   <title>LMHT</title>
   <link type='text/css' href='css/style.css' rel='stylesheet'/>
-  <link rel='stylesheet' href='js/jquery-ui-1.11.4.custom/jquery-ui.min.css'/>
+
+
+
+  <script src="https://ajax.googleapis.com/ajax/libs/jquery/1.8.3/jquery.min.js" type="text/javascript"></script>
+<script src="js/jquery-scrolltofixed-min.js" type="text/javascript"></script>
+<link rel='stylesheet' href='js/jquery-ui-1.11.4.custom/jquery-ui.min.css'/>
```
Comment (bigtiger) [inline index.html]: Indentation is incorrect^^

File: index.html [hunk 7]
```diff
@@ -29,6 +42,27 @@
...
+       </div>
+     </div>
+  </nav>
+</div>
+
+
+      
+  
+
+
```
Comment (bigtiger) [inline index.html]: Whitespace?

**Comment by bigtiger** [comment]
:+1::+1:

**Comment by ahaynes-gpsw** [comment]
Super beautiful! :+1:

**Comment by LipGlossary** [comment]
Super beautiful! :+1:

File: css/style.css [hunk 8]
```diff
@@ -1,160 +1,206 @@
-
 @import url(http://fonts.googleapis.com/css?family=Source+Sans+Pro:700,300,300italic|Roboto+Mono);
-
 * {
-  box-sizing: border-box;
-  margin: 0;
-  padding: 0;
+    box-sizing: border-box;
```
Comment (itsthatguy) [inline css/style.css:3]: Is there a reason for using 4 spaces instead of 2 spaces?

Comment (bigtiger) [inline css/style.css:3]: Because HTML-CSS-JS-Prettify and I didn't notice.

**Comment by ericanaughton** [comment]
@SaVanceX i notice something now, that we might want to discuss as a group.  the sticky nav is able to move all the way to the top of the page for the elements 'body', 'footer' and 'header'.  however, when any of the other elements are selected, the sticky nav cannot move all the way to the top. this may happen because the content is shorter, so the page isnt long enough for the sticky nav to reach the top. i think having the placement/movement of the sticky bar should be consistent for every element.  just a thought.

**Comment by ericanaughton** [comment]
@SaVanceX i realize now that my previous comment is not something we need to consider this early on.  we have a lot more redesigning to do in future sprints, where we can acknowledge that minor detail.
